### ISSUE TITLE
blacklisted table parameter use LIKE or ILIKE instead of IN

### ISSUE THREAD

**Issue by ChrisLimbik** [awslabs/amazon-redshift-utils#401]
cannot blacklist / whitelist based on prefix or suffix.
for blacklisting a table I would like to be able to add a table prefix% which matches any table name with the matching prefix or %suffix  for a matching suffix 
I.e. use the ILIKE or LIKE or SIMILAR TO clause as opposed to IN

**Comment by IanMeyers** [comment]
Yeah this makes sense. We have a regex search on schemas but not on the tables themselves. Happy to take a PR for this or perhaps can look at it over the holidays.
